[bug] Individual wallet import TS error

### Is there an existing issue for this?

- [X] I have searched the existing issues

### RainbowKit Version

0.7.0

### wagmi Version

5.6.8

### Current Behavior

Importing individual wallets from the documented path `@rainbow-me/rainbowkit/wallets` generates the following TS error:

`Cannot find module '@rainbow-me/rainbowkit/wallets' or its corresponding type declarations.ts(2307)`


### Expected Behavior

_No response_

### Steps To Reproduce

_No response_

### Link to Minimal Reproducible Example (CodeSandbox, StackBlitz, etc.)

_No response_

### Anything else?

_No response_

--- a/.changeset/gorgeous-penguins-carry.md
+++ b/.changeset/gorgeous-penguins-carry.md
@@ -1,0 +1,5 @@
+---
+'@rainbow-me/rainbowkit': patch
+---
+
+Fix package.json exports field backwards compatibility for `@rainbow-me/rainbowkit/wallets`

--- a/packages/rainbowkit/package.json
+++ b/packages/rainbowkit/package.json
@@ -4,7 +4,8 @@
   "description": "The best way to connect a wallet",
   "files": [
     "dist",
-    "styles.css"
+    "styles.css",
+    "wallets"
   ],
   "type": "module",
   "exports": {

